packages/fether-react/src/BackupPhrase/BackupPhrase.js

```diff
@@ -0,0 +1,368 @@
+
+    // phrase is still encrypted
+    this.setState({ phrase });
+  };
+
+  onCopyOrPastePhrase = event => {
+    // Disable copying the phrase if in production.
+    // Keep it enabled for development.
+    if (process.env.NODE_ENV === 'production') {
+      event.preventDefault();
+      this.setState({
+        error:
+          'Copy and pasting is disabled for this step. Please type out your full recovery phrase.'
+      });
+
+      return false;
+    }
+  };
+
+  nextStep = event => {
+    const {
+      account: { address },
+      history,
+      location: { pathname }
+    } = this.props;
+
+    event.preventDefault();
+
+    if (pathname.split('/')[3] !== 'true') {
+      history.push(`/tokens/${address}`);
+    }
+
+    this.setState({
+      step: this.state.step + 1
+    });
+  };
+
+  prevStep = () => {
+    const {
+      account: { address },
+      history
+    } = this.props;
+    const { step } = this.state;
+
+    if (step === 1) {
+      history.push(`/tokens/${address}`);
+    }
+
+    this.setState({
+      step: step - 1
+    });
+  };
+
+  // Decrypt the encrypted phrase from state
+  unlockWithPassword = async event => {
+    const { password, phrase } = this.state;
+
+    try {
+      // decrypt the phrase
+      var phraseInBytes = CryptoJS.AES.decrypt(phrase, password);
+      // this will throw in the case of a wrong password
+      var originalPhrase = phraseInBytes.toString(CryptoJS.enc.Utf8);
+
+      if (!originalPhrase) {
+        throw new Error('Incorrect password');
+      }
+
+      this.setState({
+        error: null,
+        phrase: originalPhrase,
+        unlocked: true
+      });
+    } catch (error) {
+      event.preventDefault();
+
+      this.setState({
+        error: error.message + '. \n Please check your password and try again.'
+      });
+
+      return false;
+    }
+  };
+
+  render () {
+    const {
+      account: { address, name },
+      location: { pathname }
+    } = this.props;
+    const { error, unlocked } = this.state;
+
+    const needsRewrite = pathname.split('/')[3];
+    const title =
+      needsRewrite === 'true'
+        ? 'Backup Recovery Phrase'
+        : 'View Recovery Phrase';
+
+    return (
+      <div>
+        <Header
+          left={
+            <Link className='icon -back' to='/accounts'>
+              Back
+            </Link>
+          }
+          title={<h1>{title}</h1>}
+        />
+
+        <div className='window_content'>
+          <div className='box -padded'>
+            <RequireHealthOverlay require='node'>
+              <AccountCard
+                address={address}
+                name={name}
+                drawers={[
+                  <div>
+                    {unlocked
+                      ? this.renderCopyAndRewrite()
+                      : this.renderPasswordForm()}
+                    {needsRewrite === 'true' ? this.renderTips() : null}
+                    {error}
+                    <nav className='footer-nav'>
```

Comment:
Can we move the footer down outside of the `AccountCard`, so that the health status is displayed like on any other screen.